### ISSUE TITLE
No items in character selection screen

### ISSUE THREAD

**Issue by dleistn1** [WareBare/WanezModGD#2]
Hi,

first I want to thank you for this absolute awesome mod! Incredible cool work, Mate. Now to the issue I've encountered. The character used to play the mod wears no items in character selection menu, no matter if the items are custom (Planeshifters gear) or found ones. I can attach a screen if you need it.

Another question: is it by design that there is no music in the levels? (I added music in my local version, but unfortunately it isn't tracked via Git)

Greetings,
Dan

**Comment by WareBare** [comment]
Gear not showing in the main menu is a general mod thing, nothing to do with this mod in particular. I guess because Epics/Legendaries are now as dbr inside the mod it also affects them. Nothing I can do about it 😞 .

Sounds... well, since I don't play with sounds... ever, so I have no idea where music is missing or if there should be music, also some of the DGA maps don't have music, because of that same reason. Can't speak for those made by the community though, they could have left out the music for a reason.

Thank you for the report.

**Comment by dleistn1** [comment]
Ah okay, I see. But just for your info in GD Eternal Mod are the items in char selection visible. If I got time Icould open a branch and add the music. Maybe you can merge it into master, cause I missed the music a little, so I dont need to redo it every time you update;)

**Comment by WareBare** [comment]
yeah vanilla items usually work just fine, it's possible because I had to copy the file over and change a lua hook it breaks it, or the fact that the levels.arc needs to be copied over which includes something for the mainmenu. Either way I tried fixing this issue for custom items back when the tools were released and copied a vanilla file over to make a new item, nothing changed and it still didn't show up in the mainmenu.

I'm guessing you changed the level files for music, which havn't been touched since the beginning and I don't intend to change anything in DGA for a while, just have to replace the wanez_dga_map.arc with yours until anything changes in source/wanez_dga_map/. For now I have so much other stuff on my list that I won't be touching that map for a lot of updates. It's gonna be all new masteries, items and a new level file for a while.
You can still open a branch or upload it somewhere, I just can't promise I'd be looking into it for a long time, even the next update is probably not even gonna happen next month, since it's the month of the year where RL is taking over every part of my life 😁

**Comment by dleistn1** [comment]
Sure, the same for me, Christmas is coming:) Thank you for the infos and keep up your outstanding work. Crate should definitely hire you:) 
Wish you a merry X-Mas
Dan

**Comment by WareBare** [comment]
Thanks and a Merry Christmas to you, too. 😄
